documentation/source/development/clang-format.rst

```diff
@@ -0,0 +1,17 @@
+Clang format
```

Comment:
Please link or mention the [exceptions](https://github.com/inexorgame/vulkan-renderer/blob/91b8915d5d6659e302a7cecd24177804c027e6a1/documentation/source/development/engine-design/coding-style.rst#removed-clang-tidy-checks)

Reply:
Right. We should add a sub-section "error handling".

Reply:
Done.